cpp/cmake_modules/ThirdpartyToolchain.cmake

```
@@ -4728,8 +4729,9 @@ macro(build_awssdk)
                                               "10")
     # Workaround for https://github.com/aws/aws-sdk-cpp/issues/1750
     set(AWSSDK_PATCH_COMMAND "sed" "-i.bak" "-e" "s/\"-Werror\"//g"
-                             "<SOURCE_DIR>/cmake/compiler_settings.cmake")
+                             "<SOURCE_DIR>/cmake/compiler_settings.cmake" "&&")
   endif()
+  list(APPEND AWSSDK_PATCH_COMMAND "${AWSSDK_SOURCE}/prefetch_crt_dependency.sh")
```

Can we use `<SOURCE_DIR>`?

```suggestion
  list(APPEND AWSSDK_PATCH_COMMAND "<SOURCE_DIR>/prefetch_crt_dependency.sh")
```

See https://cmake.org/cmake/help/latest/module/ExternalProject.html for `<SOURCE_DIR>`.

---

BTW, I think that we should not use this. If we use this, this code doesn't work on Windows. (Windows doesn't provide `sh`.)